### PR TITLE
probeCircles on search

### PR DESCRIPTION
**Pull request by ArtificialOwl**
search should be improved

- prefer probeCircles to getCircles
- do not search on display_name

### CODE REVIEW

**Comment by ArtificialOwl** [comment]
should fix #1033

File: lib/Db/CoreQueryBuilder.php [hunk 1]
```diff
@@ -41,6 +41,7 @@
 use OCA\Circles\Model\FederatedUser;
 use OCA\Circles\Model\Member;
 use OCA\Circles\Model\Probes\CircleProbe;
+use OCA\Circles\Model\Probes\DataProbe;
```
Comment (Altahrim) [inline lib/Db/CoreQueryBuilder.php:44]: It doesn't seem to be used

**Review by Altahrim — APPROVED** [review]

**Pull request merged**